tests/test_install.py

```diff
@@ -257,7 +257,7 @@ def _mock_fetch_bytes(arg, *args, **kwargs):
 
     msg = "Access-Control-Allow-Origin"
     with pytest.raises(ValueError, match=msg):
-        await micropip.install("htps://x.com/xxx-1.0.0-py3-none-any.whl")
```

Comment:
It's not entirely clear if the typo here was on purpose.

Reply:
Looks pretty accidental to me. If it were on purpose you would expect a comment.